resolwe_bio/tools/cuffnorm_output_table_headers.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python2
+# pylint: disable=missing-docstring,invalid-name
+# XXX: Refactor to a comand line tool and remove pylint disable
+"""Rename cuffnorm output tables headers."""
+from __future__ import absolute_import, division, print_function
+
+import argparse
+from operator import itemgetter
+
+parser = argparse.ArgumentParser(description="Rename cuffnorm output tables headers")
+parser.add_argument('replicates', help="Define replicates and groups")
+parser.add_argument('sample_names', help="Sample names")
+args = parser.parse_args()
+
+replicates = args.replicates.split(",")
+sample_names = args.sample_names.split(",")
+
+samples=[]
+#This sort must produce the same order as cufflinks_sample_groups.py
+for x in sorted(zip(replicates,sample_names), key=itemgetter(0)):
+    samples.append(x[1])
+                                            
+def replace_labels(table_name,sample_list):
+    '''
+    inputs: table_name - cuffnorm output table files, 
+            sample_list - list of sample names
+    description: function replaces original table headers which
+    contain group names with sample names
+    '''  
+    with open(table_name, 'r') as genes_table:
+        file_lines=genes_table.readlines()[1:]
```

Comment:
Use `next(genes_table)` as you did before.

`[1:]`copies the whole list, so you will read the whole file and then copy it just to get rid of first line.